docs/dev/libvirt-howto.md

```diff
@@ -286,6 +286,32 @@ kubectl get --all-namespaces pods
 ## Troubleshooting
 If following the above steps hasn't quite worked, please review this section for well known issues.
 
+### console doesn't come up
+In case of libvirt there is no wildcard dns resolution and console is depend on the route which is created by auth operator.
+To make it work we need to use the multi-stage installation and edit the `domain` for ingress.
+
+Following steps are required to have console operator up and working.
+```console
+// Add another domain entry in the openshift.conf which used by dnsmasq
+// Here `tt.testing` is the domain which I choose when running the installer.
+$ cat /etc/NetworkManager/dnsmasq.d/openshift.conf
+server=/tt.testing/192.168.126.1
+address=/.apps.tt.testing/192.168.126.51
```

Comment:
This will only work for a single worker, right?

Reply:
@gbraad yes and this is better than nothing atm.

Reply:
will it .apps.tt.testing  or .apps.test.tt.testing

Reply:
@anildhingra no it is `apps.tt.testing` not the domain which your cluster use.

Reply:
it still fails 

2019/03/11 13:33:10 auth: error contacting auth provider (retrying in 10s): request to OAuth issuer endpoint https://openshift-authentication-openshift-authentication.apps.test.tt.testing/oauth/token failed: Head https://openshift-authentication-openshift-authentication.apps.test.tt.testing: dial tcp: lookup openshift-authentication-openshift-authentication.apps.test.tt.testing on 172.30.0.10:53: no such host
2019/03/11 13:33:20 auth: error contacting auth provider (retrying in 10s): request to OAuth issuer endpoint https://openshift-authentication-openshift-authentication.apps.test.tt.testing/oauth/token failed: Head https://openshift-authentication-openshift-authentication.apps.test.tt.testing: dial tcp: lookup openshift-authentication-openshift-authentication.apps.test.tt.testing on 172.30.0.10:53: no such host

if i do dummy nslookup it works

[root@test-wm76w-master-0 ~]# nslookup one.apps.test.tt.testing
Server:		192.168.126.1
Address:	192.168.126.1#53

but in above case its looking - openshift-authentication-openshift-authentication.apps.test.tt.testing

Reply:
@anildhingra,

Did you try applying my workaround?

Regards,
Fabio Sbano

Reply:
@anildhingra you didn't perform all steps, if you check later you need to change the route also before starting the cluster.

Reply:
@praveenkumar

"+To make it work we need to use the multi-stage installation and edit the `domain` for ingress."

I understood

Reply:
yes , domain was missing - after modification its working now

Reply:
@praveenkumar,

Could not this be done automatically when libvirt is selected?

Extend the functionality of create.go

Congratulations also for the solution !! 

Best Regards,
Fábio Sbano

Reply:
This is a temporary solutin. In future we do not want to have to do so many modifications. Since a solution that is cross-platform is preferred, we might not continue with using dnsmasq, but rather coredns for example.

Reply:
@gbraad,

I solved with bind on the physical machine without having to change the (route). #1397

Installer exited by timeout (30min) and I found problem too #1394

(I do not know if it was due to this workaround)

Best Regards,
Fábio Sbano

Reply:
@praveenkumar This worked for me, but please explicitly mention the the IP address is on the worker node

Reply:
I can't find the manifest to edit in my cluster's installation directory:

```
steveej@steveej-t480s-work ✓ ~/src/job-redhat/GOPATH/src/github.com/openshift/installer/openshift-steveej  954533fcc ⚡
$ tree                                      
.
├── auth
│   ├── kubeadmin-password
│   └── kubeconfig
├── disable-bootstrap.tfvars
├── metadata.json
├── terraform.tfstate
└── tls
    ├── journal-gatewayd.crt
    └── journal-gatewayd.key
```

How do I get the installer to expose them?